tests/quick.rs

```diff
@@ -109,6 +111,15 @@ fn do_ops<K, V, S>(
                 evmap.empty(k.clone());
                 write_ref.remove(k);
             }
+            RemoveValue(ref k, ref v) => {
+                evmap.remove(k.clone(), v.clone());
+                write_ref.get_mut(k).and_then(|values| {
+                    values
+                        .iter_mut()
+                        .position(|value| value == v)
+                        .and_then(|pos| Some(values.remove(pos)))
```

Comment:
I guess I could `swap_remove` to speed up the tests since we sort later anyway

Reply:
Oh, yeah, that'd be great! Unlikely to matter _too_ much though.